proposals/NNNN-Vehicle-data-additions.md

```diff
@@ -0,0 +1,1341 @@
+# Vehicle Data Additions
+
+* Proposal: [SDL-NNNN](NNNN-Vehicle-data-additions.md)
+* Author: [Ankur Tiwari](https://github.com/atiwari9)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core | HMI | SDL SERVER | SHAID | PROXY]
```

Comment:
Please change "PROXY" to specific platforms.  For example, Java Suite, iOS, JavaScript Suite.